app/assets/javascripts/components/high_order/editable.cjsx

```diff
@@ -28,13 +28,14 @@ Editable = (Component, Stores, Save, GetState, Label) ->
       if permissions && @state.editable
         unless save_only
           className = 'controls'
-          cancel = (
-            <button onClick={@cancelChanges} className='button'>{I18n.t('editable.cancel')}</button>
-          )
+          unless SaveOnly
```

Comment:
To clarify... the original save_only option seems to be incompatible with the layout of the other controls. Using that instead of adding the new one resulted in shifting the position of the other controls, and so I opted for adding new logic that I knew would not affect any other usages of Editable in other contexts.